specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -3648,6 +3648,240 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualHubs/{virtualHubName}/hubRouteTables/{routeTableName}": {
+      "put": {
+        "operationId": "RouteTables_CreateOrUpdate",
+        "x-ms-examples": {
+          "RouteTablePut": {
+            "$ref": "./examples/RouteTablePut.json"
+          }
+        },
+        "description": "Creates a RouteTable resource if it doesn't exist else updates the existing RouteTable.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VirtualHub."
+          },
+          {
+            "name": "virtualHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VirtualHub."
+          },
+          {
+            "name": "routeTableName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the RouteTable."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "RouteTableParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RouteTable"
+            },
+            "description": "Parameters supplied to create or update RouteTable."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of the RouteTable created or updated.",
+            "schema": {
+              "$ref": "#/definitions/RouteTable"
+            }
+          },
+          "201": {
+            "description": "Request received successfully. Returns the details of the RouteTable created or updated.",
+            "schema": {
+              "$ref": "#/definitions/RouteTable"
+            }
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      },
+      "get": {
+        "operationId": "RouteTables_Get",
```

Comment:
@dedhar : i might need to keep the name of the files for the examples as HubRouteTable because there are already existing files named RouteTable